src/test/java/bio/terra/service/resourcemanagement/BufferServiceConnectedTest.java

```diff
@@ -82,16 +89,19 @@ public void setup() throws Exception {
 
   @After
   public void tearDown() throws Exception {
+    for (String googleProjectId : googleProjectIds) {
+      resourceManagerService.cloudResourceManager().projects().delete(googleProjectId).execute();
```

Comment:
It would be nice to do this delete as a part of a flight so we don't have a bunch of orphaned projects (definitely a separate ticket's worth of work)

Reply:
Also I thought the projects from this test would get cleaned up in the tools environment?

Reply:
This connected test doesn't run a flight.